Dokan/Dokan/Scenes/Registration/RegisterViewController.xib

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please update the constraints to match the updated design here https://www.figma.com/file/qlqfcxV2osGhV6xIHSoatM/E-Commerce-Mobile-Apps-%231-(Community)?node-id=16%3A3680